### ISSUE TITLE
Investigate possibility of easing the workflow of adding/removing ESI nodes from an OpenShift cluster

### ISSUE THREAD

**Issue by tzumainn** [CCI-MOC/esi#612]

**Comment by ajamias** [comment]
Would an esi-ui implementation sound nice?

**Comment by tzumainn** [comment]
Yep, but the underlying logic needs to work first, presumably placed in esisdk. So we'd have to work out the OpenShift REST API and the process of getting/passing the tokens, etc. This is partially described in https://esi.readthedocs.io/en/latest/usage/openshift.html, but some formalization will be needed.
